README.md

```diff
@@ -1 +1,69 @@
-# microverse-portfolio
\ No newline at end of file
+# Microverse-portfolio
+
+>This project is about creating a portfolio page using best practices for HTML and CSS and uses linters, GitHub Flow rules and documentation of work. It is important to start with a new repository and setup the linters, follow a design template for the portfolio or create your own through Figma and code it using HTML and CSS. I built the mobile website using a Figma template provided by Microverse. I used the linters - GitHub Actions, Lighthouse, Webhint and Stylelint to make this project error-free. Also, all the other files except the HTML and CSS files were added to feature branches to make the main branch remain in best possible condition and stable.
```

Comment:
- [ ] Please make sure to include the Table of content from the Microvere README Template in your README. Also please link each section with the Table of Content so that when a user clicks on a particular section in the table of content it takes it to that section.